installers/beamsim-jupyter/radiasoft-download.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+#
+# Install dependencies common to beamsim-jupyter and container-jupyter-nvidia but
+# install the correct version for gpu vs not.
+#
+
+beamsim_jupyter_main() {
+    declare gpu_only=${1:-}
```

Comment:
For flags like this, it's annoying. I think instead you should have something like beamsim_jupyter_gpu=1. This way the code is documented. It's not "gpu_only" because the other is "cpu_only". It's gpu or not gpu, so you don't need the only. Alternatively, you could have:

```sh
declare mode=$1
case $mode in
gpu)
cpu)
*)
install_err
```

There's no need to take shortcuts here especially with installers, since there's only a little bit of code needed and it documents the intention of something we rarely visit.

Reply:
`gpu_only` is what you recommended in the past https://github.com/radiasoft/download/pull/187#discussion_r770979574. This case seemed similar to me so rather than invent a new concept I thought it made sense to stick with what we already have.  "shortcut" never crossed my mind...

Reply:
> elegant_install_bin "$gpu_only" (which, since internal, can just be a 1 or empty).

The emphasis is on "since internal". This is all within one file. The parameter in this case is being passed from one installer to another.

Reply:
`gpu-only` is set by another installer https://github.com/radiasoft/container-jupyter-nvidia/blob/522bc38178df77576b493e783bfece1d50ae0099/container-conf/build.sh#L36

Reply:
gpu-only is not correct if it can't be used on a cpu. Is that the case here?

Reply:
I hate to say this but that double negative is making your sentence hard to understand. Can you say it a different way?

Reply:
In this case is "gpu-only" meaning the other case is "gpu and cpu" or is gpu-only meaning "gpu instead of cpu"? I think the flag gpu-only may be incorrect, IOW.